Update azure-security-disk-encryption-tsg.md

ejarvi opened the pull request:
Add Docker to list of applications that are not currently compatible with OS disk encryption logic

azurecla commented:
Hi __@stuartleeks__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

ejarvi committed:
Update azure-security-disk-encryption-tsg.md

Add Docker to list of applications that are not currently compatible with OS disk encryption logic
